static/js/drafts.js

```diff
@@ -391,10 +400,56 @@ export function launch() {
         return sorted_formatted_drafts;
     }
 
-    function render_widgets(drafts) {
+    function render_widgets(narrow_drafts, other_drafts) {
         $("#drafts_table").empty();
+
+        let narrow_drafts_header;
+        // Compose box headers get priority.
+        // TODO: encode this somehow to stay in sync with how filter_drafts_by_compose_box_and_recipient works.
+        if (compose_state.get_message_type() === "stream" && compose_state.topic()) {
+            narrow_drafts_header = $t(
+                {defaultMessage: "Drafts from #{stream} > {topic}"},
```

Comment:
For these strings, I wonder if it'd make sense to just have a single translation string of the form `Drafts from {recipient}`, and just have these blocks each set `recipient`.

Reply:
That makes sense! Sorry if I'm missing it in https://github.com/zulip/zulip/blob/main/docs/translating/internationalization.md, but how can I write the translation string and insert the variable after?

Reply:
You need to just get `recipient`  in these if conditions, the set just return 

```js
$t(
        {defaultMessage: "Drafts from {recipient}"},
        {recipient: recipient},
);
```

Reply:
ah thank you!!